conda-recipes/meta.yaml

```
@@ -0,0 +1,33 @@
+{% set VERSION_SUFFIX = "" %} # debug version suffix, appended to the version
+
+package:
+  name: numba_rvsdg
+  # GIT_DESCRIBE_TAG may not be set
+  version: {{ "%s%s" % (environ.get('GIT_DESCRIBE_TAG', '').lstrip('v'), VERSION_SUFFIX) }}
```

`numba-rvsdg` only has a single tag so far and it doesn't have a `v` prefix. Hence I reckon this line should be:

```suggestion
  version: {{ "%s%s" % (environ.get('GIT_DESCRIBE_TAG', ''), VERSION_SUFFIX) }}
```